Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/tests-dashboard/test-dashboard.pipe.ts

```diff
@@ -1,17 +1,19 @@
-﻿import { Pipe, PipeTransform } from '@angular/core'
-@Pipe({
+﻿import { Pipe, PipeTransform } from '@angular/core';
+import { Test } from '../tests.model';
 
+@Pipe({
     name: 'filter'
-
 })
 export class FilterPipe implements PipeTransform {
 
-    transform(value: any, term: any): any {
-        if (term === undefined)
-            return value;
-        return value.filter(function (x: any) {
-
-            return (x.testName.toLowerCase().includes(term.toLowerCase()) || x.link.toLowerCase().includes(term.toLowerCase()));
-        })
+    transform(allTests: Test[], searchedTest: string) : any {
+        if (searchedTest === undefined)
+            return allTests;
+        return allTests.filter(function (currentTest: Test) {
+            if (currentTest.link === null)
```

Comment:
So, I think search won't be performed on test link. It will be only with Test Name. Is there anything written in requirement to perform search at Test Link also?